### PR TITLE
alpine docker: update GH action versions

### PR DESCRIPTION
**Pull request by neteler**
update GH action versions

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -8,11 +8,11 @@ jobs:
 
   docker:
     name: build and push
-    runs-on: ubuntu-latest
+    runs-on: s1tbx-latest
```
Comment (neteler) [inline .github/workflows/docker.yml]: As it looked like a typo to me I have changed this line. Pls check @mmacata if that's right.

Comment (mmacata) [reply .github/workflows/docker.yml]: This is a coincidence as our other branch is named like this. It is not a typo as this image referrs to where the build runs and the images are provided by github. We use ubuntu-latest to run the build, in this case an alpine build. Docker in docker :upside_down_face:

Comment (neteler) [inline .github/workflows/docker.yml]:
```suggestion
    runs-on: ubuntu-latest
```

Bringing back original name.

**Review by mmacata — APPROVED** [review]

File: .github/workflows/docker.yml (hunk 1 above)
Comment (neteler) [reply .github/workflows/docker.yml]: I see, reverted in c8ac7e0

**Pull request merged**